cpp/src/hdbscan/runner.h

```diff
@@ -56,7 +57,7 @@ struct FixConnectivitiesRedOp {
   FixConnectivitiesRedOp(value_idx* colors_, value_t* core_dists_, value_idx m_)
     : colors(colors_), core_dists(core_dists_), m(m_){};
 
-  typedef typename cub::KeyValuePair<value_idx, value_t> KVP;
+  typedef typename raft::KeyValuePair<value_idx, value_t> KVP;
```

Comment:
Generally, we should prefer [using over typedef](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rt-using) for this sort of thing, but given the time constraints, this is fine for now.

Reply:
Yeah, we've been doing that in general but I was trying to minimize the number of changes here. Unfortunately I didn't build the benchmarks locally when testing so the first attempt failed in a redundant benchmark.